Microsoft.Xrm.Data.PowerShell/Microsoft.Xrm.Data.PowerShell.psm1

```diff
@@ -1901,52 +1901,56 @@ function Import-CrmSolutionAsync{
 		catch
 		{
 			throw LastCrmConnectorException($conn)
-		}    
+		}
+
         $pollingStart = Get-Date
         $isProcessing = $true
 		$secondsSpentPolling = 0
-        $pollingDelaySeconds = 5
+        $pollingDelaySeconds = 60
```

Comment:
Would you add an optional parameter to the function for the polling delay? Short polling delays are nice for better logging and granularity for measuring import times and process etc.  longer by default is fine but the ability to have it shorter is nice as well.

Reply:
Yep, better as a param.